### PR TITLE
helper.py: Change the helper message for nolaunch and noupdate

### PR DESCRIPTION
**Pull request by tushar-rishav**
Fixes https://github.com/coala-analyzer/coala-html/issues/59

### CODE REVIEW

File: coalahtml/helper.py [hunk 1]
```diff
@@ -96,5 +96,5 @@ def get_args():
                             const=True,
                             metavar='BOOL',
                             default=False,
-                            help='Launch webpage')
+                            help='Do not launch webpages.')
```
Comment (sils1297) [inline coalahtml/helper.py]: webpages -> webserver

**Comment by sils1297** [comment]
ack https://github.com/coala-analyzer/coala-html/pull/61/commits/62eae75902a85e7c7920936ad550eb851ebc1bdd

File: coalahtml/helper.py [hunk 2]
```diff
@@ -84,7 +84,7 @@ def get_args():
                             const=True,
                             metavar='BOOL',
                             default=False,
-                            help='Launch webpage from existing results.')
+                            help='Launch webserver from existing results.')
```
Comment (tushar-rishav) [inline coalahtml/helper.py]: Ah damn. Now I would have to change commit message to include --noupdate too :P

**Comment by tushar-rishav** [comment]
reack c747694

**Comment by tushar-rishav** [comment]
@rultor merge

**Pull request merged**